articles/active-directory/develop/quickstarts/active-directory-android.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Azure AD v2 Android quickstart | Microsoft Docs
+description: Learn how Android native applications can call an API that requires access tokens by Azure Active Directory v2.0 endpoint.
```

Comment:
Let's remove `native` from this description.

Reply:
Is it worth highlighting the types of users this will show them how to sign in (business or consumer)

Reply:
@danieldobalian - as we migrate this to docs, we will make sure that every page that applies to v1 or v2 only have a header to say so